src/Kiota.Builder/Writers/Go/CodeMethodWriter.cs

```diff
@@ -110,8 +110,10 @@ private void WriteRequestBuilderBody(CodeClass parentClass, CodeMethod codeEleme
                 WriteReturnError(writer);
                 shouldDeclareErrorVar = false;
             }
-            foreach(var otherProp in parentClass.GetPropertiesOfKind(CodePropertyKind.Custom))
-                WriteSerializationMethodCall(otherProp.Type, parentClass, otherProp.SerializationName ?? otherProp.Name.ToFirstCharacterLowerCase(), $"m.Get{otherProp.Name.ToFirstCharacterUpperCase()}()", shouldDeclareErrorVar, writer);
+            foreach(var otherProp in parentClass.GetPropertiesOfKind(CodePropertyKind.Custom)) {
+                var accessorName = otherProp.IsNameEscaped && !string.IsNullOrEmpty(otherProp.SerializationName) ? otherProp.SerializationName : otherProp.Name.ToFirstCharacterLowerCase();
```

Comment:
I don't think this is blocker as I have searched and haven't spotted any instance where the `SerializationName` is set and in the instance of setting it is set with an uppercase letter hence needing the .ToFirstCharacterLowerCase() call (like other instances in the PR)